File: Contest Natal 22/c - Dividindo Presentes.cpp
#include <bits/stdc++.h>
using namespace std;
int main()
{
    long long a,b;
    cin >> a >> b;
    if(a==b)cout<<"Sobraram 0 presentes!\n";
    else if (a<b) cout<<"Ainda faltam "<<fabs(a-b)<<" presentes!\n";
    else cout<<"Sobraram "<<fabs(a%b)<<" presentes!\n";
   return 0;
}

File: Contest Natal 22/c - Dividindo Presentes.py
a = int(input())
b = int(input())
c = a mod b
if  a==b:
    print("Sobraram 0 presentes!\n")
elif a<b:
    print("Ainda faltam "+abs(a-b)+" presentes!\n")
else:
    print("Sobraram "+c+" presentes!\n")